### ISSUE TITLE
Using ground truth masks instead of the predicted ones

### ISSUE THREAD

**Issue by goncayilmaz** [facebookresearch/Mask2Former#221]
Hello all,

I wanted to try something so I take the maskformer mask_cls_results and instead of using the mask_pred_results, I want to use the ground truth masks. However, the metric is decreasing a lot instead of increasing. Could you tell me what might be wrong?

      mask_cls_results = outputs["pred_logits"] # [4, 100, 150]
      # mask classification target
      gt_instances = [x["instances"].to(self.device) for x in batched_inputs]
      targets = self.prepare_targets(gt_instances, images)
      masks = [x["masks"].unsqueeze(0).to(self.device) for x in targets]
      # print(len(masks)) 4
      # print(masks[0].shape) # [7, 640, 896]
      masks = [torch.cat([x, torch.zeros(1, (100-x.size(1)), x.size(2), x.size(3)).to(self.device)], dim=1) for x in masks]
      mask_pred_results = torch.cat(masks, dim=0)
      # print(mask_pred_results.shape) # [4, 100, 640, 896]

Note: it gives me error if number of instances is not 100
